apps/fluent-tester/src/FluentTester/theme/ThemePickers.tsx

```diff
@@ -41,8 +53,24 @@ export const PartPicker: React.FunctionComponent<PartPickerProps> = (props: Part
     },
     [setValue, onChange],
   );
-  return (
-    <Picker selectedValue={value} style={themePickerStyles.dropdown} onValueChange={onValueChange}>
+
+  const themedDropdownStyles = getThemedDropdownStyles(useTheme());
+  const dropdownIconColor = useTheme().colors.buttonIcon;
+
+  return isAndroid ? (
+    <Picker
+      selectedValue={value}
+      style={themedDropdownStyles.dropdown}
+      onValueChange={onValueChange}
+      dropdownIconColor={dropdownIconColor}
+      mode="dropdown"
+    >
+      {contents.map((entry: PartPickerEntry, index: number) => (
+        <Picker.Item label={entry.label} value={entry.value} key={`entry${index}`} />
+      ))}
+    </Picker>
+  ) : (
+    <Picker selectedValue={value} style={themedDropdownStyles.dropdown} onValueChange={onValueChange}>
```

Comment:
I know this is a minor difference from the original file, but since macOS and iOS are already split out to platform specific files, could we also do so with the android one?

Reply:
done 💯